platform/darwin/src/MGLTypes.h

```diff
@@ -3,6 +3,14 @@
 #pragma once
 
 #if TARGET_OS_IPHONE
+@class UIImage;
```

Comment:
👍

(Sorry, for some reason I thought we had already defined MGLImage, but in fact it was MGLColor that we implemented.)